app/views/videos/show.html.haml

```diff
@@ -1,9 +1,38 @@
-.row
-  %div{:class => 'col-md-7'}
-    = image_tag @video.large_cover_url
-  %div{:class => 'col-md-5'}
-    %h2
-      %strong= @video.title
-    %hr
-    %p= @video.description
+%article.video
+  .container  
+    .row
+      %div{:class => 'col-md-7'}
```

Comment:
You can also just do `.col-md-7` instead of `%div{:class => 'col-md-7'}`. `.` means class and if you don't specify a tag type then it defaults to `div`.